runtime/lua/vim/treesitter/query.lua

```diff
@@ -127,25 +130,38 @@ function M.add_predicate(name, handler, force)
   predicate_handlers[name] = handler
 end
 
+local function xor(a, b)
```

Comment:
we might have xor "natively" via the bits module (which always should be available, jit or not)

Reply:
Hmmm... Does not work with boolean ?

Reply:
Is there a list of the available modules somewhere ?